packages/react/src/Delay.tsx

```diff
@@ -1,17 +1,21 @@
-import { type PropsWithChildren, useContext, useState } from 'react'
+import { type PropsWithChildren, type ReactNode, useContext, useState } from 'react'
 import { DelayDefaultPropsContext } from './contexts'
 import { useTimeout } from './hooks'
 
 export interface DelayProps extends PropsWithChildren {
   ms?: number
+  /**
+   * @experimental This is experimental feature.
+   */
+  fallback?: ReactNode
 }
 
-export const Delay = ({ ms, children }: DelayProps) => {
+export const Delay = ({ ms, children, fallback }: DelayProps) => {
   const defaultProps = useContext(DelayDefaultPropsContext)
   const delayMs = ms ?? defaultProps.ms ?? 0
   const [isDelayed, setIsDelayed] = useState(delayMs === 0)
```

Comment:
Shouldn't we think about using the `Delay` component in the Server Component? 
Of course, our `tsup` config file seems to have 'use client' attached to everything, but I just wanted to ask just to be sure!

Reply:
Could you more explain?

Reply:
You shouldn't even bother with this review anymore! 
I was wrong to leave it.